tyche-update-helper.php

```diff
@@ -34,3 +51,41 @@ function ts_modify_curl_request( &$handle, $args, $url ) {
 }
 
 add_action( 'http_api_curl', 'ts_modify_curl_request', 10, 3 );
+
+/**
+ * Function orddd_remove_transient_from_rest_api.
+ *
+ * @version 1.1.0
+ */
+function orddd_remove_transient_from_rest_api() {
+	$orddd_is_transient_removed = get_option('orddd_is_transient_removed');
+	$acpro_is_transient_removed = get_option('acpro_is_transient_removed');
+	$is_orddd_plugin_active     = is_plugin_active( 'order-delivery-date/order_delivery_date.php' );
+	$is_acpro_plugin_active     = is_plugin_active( 'woocommerce-abandon-cart-pro/woocommerce-ac.php' );
+
+	// This will remove the unwanted transient from Order Delivery Date Pro for WooCommerce plugin.
+	if ( ! $orddd_is_transient_removed && $is_orddd_plugin_active ) {
+		$fname   = ABSPATH . '/wp-content/plugins/order-delivery-date/plugin-updates/EDD_SL_Plugin_Updater.php';
+		$fhandle = fopen( $fname, "r" );
+		$content = fread( $fhandle, filesize( $fname ) );
+		$content = str_replace( '&& ! $plugin_transient', '', $content );
+		$fhandle = fopen( $fname, "w" );
+		fwrite( $fhandle, $content );
+		fclose( $fhandle );
+		update_option( 'orddd_is_transient_removed', 'yes' );
+	}
+
+	// This will remove the unwanted transient from Abandoned Cart Pro for WooCommerce plugin.
+	if ( ! $acpro_is_transient_removed && $is_acpro_plugin_active ) {
+		$fname   = ABSPATH . '/wp-content/plugins/woocommerce-abandon-cart-pro/plugin-updates/EDD_AC_WOO_Plugin_Updater.php';
+		$fhandle = fopen( $fname, "r" );
+		$content = fread( $fhandle, filesize( $fname ) );
+		$content = str_replace( '&& ! $plugin_transient', '', $content );
+		$fhandle = fopen( $fname, "w" );
+		fwrite( $fhandle, $content );
+		fclose( $fhandle );
+		update_option( 'acpro_is_transient_removed', 'yes' );
+	}
+}
+
+add_action( 'admin_init', 'orddd_remove_transient_from_rest_api' );
```

Comment:
Please use related and meaningful names.

its ideal if function names are prefixed with the initials of the current plugin. Suggestion: `tyh_remove_transient_code`